### PR TITLE
[VarDumper] Support PHPUnit --colors option

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Bug fix?      | no
| New feature?  | yes 
| Deprecations? | no 
| Tickets       | N/A 
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/VarDumper/Dumper/CliDumper.php [hunk 1]
```diff
@@ -522,12 +522,14 @@ protected function supportsColors()
                         case '--color=yes':
                         case '--color=force':
                         case '--color=always':
+                        case '--colors=always':
```
Comment (fabpot) [inline src/Symfony/Component/VarDumper/Dumper/CliDumper.php:525]: Does --colors also support yes and force as a value?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/VarDumper/Dumper/CliDumper.php [hunk 2]
```diff
@@ -522,12 +522,14 @@ protected function supportsColors()
...
                             return static::$defaultColors = true;
 
                         case '--no-ansi':
                         case '--color=no':
                         case '--color=none':
                         case '--color=never':
+                        case '--colors=never':
```
Comment (fabpot) [inline src/Symfony/Component/VarDumper/Dumper/CliDumper.php:532]: Same here, I think it supports all possible values as well.

File: src/Symfony/Component/VarDumper/Dumper/CliDumper.php (hunk 1 above)
Comment (ogizanagi) [reply src/Symfony/Component/VarDumper/Dumper/CliDumper.php:525]: Nope, it doesn't: https://phpunit.readthedocs.io/en/9.3/textui.html?highlight=--colors#command-line-options

The only three values supported are `auto`, `never` and `always`:
https://github.com/sebastianbergmann/phpunit/blob/19e27be279ab0056984a42f93dace950d9116ba7/src/TextUI/DefaultResultPrinter.php#L65

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ogizanagi.

**Pull request merged**